src/submit.rs

```diff
@@ -589,6 +589,7 @@ impl<'a> Submitter<'a> {
         ring_addr: u64,
         ring_entries: u16,
         bgid: u16,
+        flags: u16,
```

Comment:
I don't want to break api, here we need to add a new method and mark the old one as deprecated.

Reply:
Done `_with_flags` now